src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,38 @@
+import unpaidIcon from '../../assets/images/paymentrequest_status_unpaid.svg';
+import paidIcon from '../../assets/images/paymentrequest_status_paid.svg';
+import partiallyPaidIcon from '../../assets/images/paymentrequest_status_partial.svg';
+
+interface StatusBadgeProps {
+  status: 'UNPAID' | 'PARTIAL' | 'PAID';
```

Comment:
Seems odd to have to send a prop all in capital letters. If we add this to the other comment, the prop value can become lowercased :D